@here/harp.gl/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "@here/harp.gl",
+    "version": "0.1.0",
+    "description": "Functionality needed to render a map.",
```

Comment:
May I suggest using a different description? "JS Bundle containing all the functionality needed to render a map with harp.gl"

Reply:
Thanks for spotting and suggestion
This was copy-pasta.